lib/processors/minifier.js

```diff
@@ -0,0 +1,79 @@
+const path = require("path");
+const terser = require("terser");
+const Resource = require("@ui5/fs").Resource;
+/**
+ * Preserve comments which contain:
+ * <ul>
+ *  <li>copyright notice</li>
+ *  <li>license terms</li>
+ *  <li>"@ui5-bundle"</li>
+ *  <li>"@ui5-bundle-raw-include"</li>
+ * </ul>
+ *
+ * @type {RegExp}
+ */
+const copyrightCommentsAndBundleCommentPattern = /copyright|\(c\)(?:[0-9]+|\s+[0-9A-za-z])|released under|license|\u00a9|^@ui5-bundle-raw-include |^@ui5-bundle /i;
+const debugFileRegex = /((?:\.view|\.fragment|\.controller|\.designtime|\.support)?\.js)$/;
+
+/**
+ * Minifies the supplied resources.
+ *
+ * @public
+ * @alias module:@ui5/builder.processors.uglifier
+ * @param {object} parameters Parameters
+ * @param {module:@ui5/fs.Resource[]} parameters.resources List of resources to be processed
+ * @returns {Promise<object>} Promise resolving with object of resource, dbgResource and sourceMap
+ */
+module.exports = async function({resources}) {
+	return Promise.all(resources.map(async (resource) => {
+		const dbgPath = resource.getPath().replace(debugFileRegex, "-dbg$1");
+		const dbgResource = await resource.clone();
+		dbgResource.setPath(dbgPath);
+
+		const filename = path.posix.basename(resource.getPath());
+		if (filename.endsWith(".support.js")) {
```

Comment:
AFAIK this is the reason why it should not be done
```suggestion
		// Support rules should not be minified to have readable code in the Support Assistant
		if (filename.endsWith(".support.js")) {
```

Reply:
Done, moved to glob pattern in Application- and LibraryBuilder